apps/condo/domains/resident/schema/ServiceConsumer.js

```diff
@@ -56,6 +56,15 @@ const ServiceConsumer = new GQLListSchema('ServiceConsumer', {
             type: Text,
             isRequired: true,
         },
+
+        organization: {
+            schemaDoc: 'Organization to which this service consumer pays to object',
```

Comment:
`models.CASCADE`? Hmmm 🤔 Sounds like it's ORGANIZATION_OWNED_FIELD

Reply:
I think we should be able to update the organization, but not quite sure. If so - this is really ORGANIZATION OWNED FIELD